<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1429</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/icons/icons/edit.512x512.png</normalon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	border: none;
	background-color: transparent;
	background: none;
	margin: 0;
	padding: 0;
	color: black;
}

#MainGUI{
	background-color: #272727;
}

#SubMenu{
	background-color: #eaeff5;
}

QToolTip{
	background-color: #eaeff5; color: black;
}

#SubMenu QPushButton{
	font: 12pt &quot;Calibri&quot;;
	text-align: left;
	padding: 10px 20px;
}

#SubMenu QPushButton:Hover{
	background-color: #d4d8e2;
}

#SubMenu QPushButton:Focus{
	background-color: #c5cad6;
	border-left: 5px solid #000;
	font-size: 12.5pt;
		/*border-top-left-radius: 32px;
	border-bottom-left-radius: 32px;*/
}

#Content QHeaderView::section{background-color: #a2a6ae;border-radius:3px;font-size: 12pt;}
#Content QTableWidget::item{background-color: #c5cad6;}
#Content QTableWidget::item:Alternate{background-color: #a2a6ae;}
#Content QTableWidget::item:Hover{background-color: #d4d8e2;}

#Content QTreeView::item{background-color: #c5cad6;}
#Content QTreeView::item:Alternate{background-color: #a2a6ae;}
#Content QTreeView::item:Hover{background-color: #d4d8e2;}</string>
  </property>
  <property name="iconSize">
   <size>
    <width>45</width>
    <height>45</height>
   </size>
  </property>
  <widget class="QWidget" name="MainGUI">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QWidget" name="Body" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #c5cad6;</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QStackedWidget" name="Content">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="HomePage">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QFrame" name="frame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <widget class="QFrame" name="frame_3">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>1171</width>
                <height>80</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <widget class="QFrame" name="frame_4">
               <property name="geometry">
                <rect>
                 <x>1030</x>
                 <y>0</y>
                 <width>131</width>
                 <height>80</height>
                </rect>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <widget class="QPushButton" name="pushButton">
                <property name="geometry">
                 <rect>
                  <x>20</x>
                  <y>30</y>
                  <width>91</width>
                  <height>30</height>
                 </rect>
                </property>
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </widget>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="MenuPage">
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>270</x>
             <y>230</y>
             <width>221</width>
             <height>19</height>
            </rect>
           </property>
           <property name="text">
            <string>Menu Page</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="AdminPage"/>
         <widget class="QWidget" name="FindPage">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QTableWidget" name="tableWidget">
                <property name="focusPolicy">
                 <enum>Qt::NoFocus</enum>
                </property>
                <property name="sizeAdjustPolicy">
                 <enum>QAbstractScrollArea::AdjustToContents</enum>
                </property>
                <property name="autoScroll">
                 <bool>false</bool>
                </property>
                <property name="editTriggers">
                 <set>QAbstractItemView::NoEditTriggers</set>
                </property>
                <property name="tabKeyNavigation">
                 <bool>false</bool>
                </property>
                <property name="alternatingRowColors">
                 <bool>true</bool>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::NoSelection</enum>
                </property>
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectItems</enum>
                </property>
                <property name="verticalScrollMode">
                 <enum>QAbstractItemView::ScrollPerPixel</enum>
                </property>
                <property name="showGrid">
                 <bool>true</bool>
                </property>
                <property name="gridStyle">
                 <enum>Qt::SolidLine</enum>
                </property>
                <property name="columnCount">
                 <number>0</number>
                </property>
                <attribute name="horizontalHeaderMinimumSectionSize">
                 <number>50</number>
                </attribute>
                <attribute name="horizontalHeaderDefaultSectionSize">
                 <number>50</number>
                </attribute>
                <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                 <bool>true</bool>
                </attribute>
                <attribute name="horizontalHeaderStretchLastSection">
                 <bool>false</bool>
                </attribute>
                <attribute name="verticalHeaderMinimumSectionSize">
                 <number>30</number>
                </attribute>
                <attribute name="verticalHeaderStretchLastSection">
                 <bool>false</bool>
                </attribute>
                <row>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="textAlignment">
                  <set>AlignCenter</set>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string/>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string/>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string/>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string/>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string/>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string/>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string/>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string/>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string/>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string/>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string/>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string/>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string/>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string/>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string/>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string/>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string/>
                 </property>
                </row>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="FlowsPage">
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>250</x>
             <y>240</y>
             <width>191</width>
             <height>19</height>
            </rect>
           </property>
           <property name="text">
            <string>Flows Page</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="OptionsPage">
          <widget class="QPushButton" name="LightBtn">
           <property name="geometry">
            <rect>
             <x>-10</x>
             <y>0</y>
             <width>191</width>
             <height>61</height>
            </rect>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>  Dark Switch</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/resources.qrc">
             <normaloff>:/darkicon/icons/dark/toggle-off.512x342.png</normaloff>:/darkicon/icons/dark/toggle-off.512x342.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="HelpPage">
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>250</x>
             <y>250</y>
             <width>161</width>
             <height>19</height>
            </rect>
           </property>
           <property name="text">
            <string>Help Page</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="Menu" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignLeft">
        <widget class="QWidget" name="SubMenu" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="MenuFrame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="HomeBtn">
               <property name="focusPolicy">
                <enum>Qt::ClickFocus</enum>
               </property>
               <property name="toolTip">
                <string>Home</string>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>     Home</string>
               </property>
               <property name="icon">
                <iconset resource="../resources/resources.qrc">
                 <normaloff>:/darkicon/icons/dark/estate.461x512.png</normaloff>:/darkicon/icons/dark/estate.461x512.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>48</width>
                 <height>48</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item alignment="Qt::AlignTop">
           <widget class="QFrame" name="HomeFrame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>50</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>20</number>
             </property>
             <item>
              <widget class="QPushButton" name="MenuBtn">
               <property name="focusPolicy">
                <enum>Qt::ClickFocus</enum>
               </property>
               <property name="toolTip">
                <string>Menu</string>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>     Menu   </string>
               </property>
               <property name="icon">
                <iconset resource="../resources/resources.qrc">
                 <normaloff>:/darkicon/icons/dark/menu.svg</normaloff>:/darkicon/icons/dark/menu.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>48</width>
                 <height>48</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="AdminBtn">
               <property name="focusPolicy">
                <enum>Qt::ClickFocus</enum>
               </property>
               <property name="toolTip">
                <string>Administrator Account</string>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>     Admin</string>
               </property>
               <property name="icon">
                <iconset resource="../resources/resources.qrc">
                 <normaloff>:/darkicon/icons/dark/user-circle.511x512.png</normaloff>:/darkicon/icons/dark/user-circle.511x512.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>48</width>
                 <height>48</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="FindBtn">
               <property name="focusPolicy">
                <enum>Qt::ClickFocus</enum>
               </property>
               <property name="toolTip">
                <string>Find Users / Materials</string>
               </property>
               <property name="text">
                <string>     Find</string>
               </property>
               <property name="icon">
                <iconset resource="../resources/resources.qrc">
                 <normaloff>:/darkicon/icons/dark/search.512x512.png</normaloff>:/darkicon/icons/dark/search.512x512.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>48</width>
                 <height>48</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="FlowsBtn">
               <property name="focusPolicy">
                <enum>Qt::ClickFocus</enum>
               </property>
               <property name="toolTip">
                <string>See the current flow of materials</string>
               </property>
               <property name="text">
                <string>    Flows    </string>
               </property>
               <property name="icon">
                <iconset resource="../resources/resources.qrc">
                 <normaloff>:/darkicon/icons/dark/location-arrow.512x512.png</normaloff>:/darkicon/icons/dark/location-arrow.512x512.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>48</width>
                 <height>48</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QFrame" name="OptionsFrame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>20</number>
             </property>
             <item>
              <widget class="QPushButton" name="OptionsBtn">
               <property name="focusPolicy">
                <enum>Qt::ClickFocus</enum>
               </property>
               <property name="toolTip">
                <string>View the settings</string>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>    Options   </string>
               </property>
               <property name="icon">
                <iconset resource="../resources/resources.qrc">
                 <normaloff>:/darkicon/icons/dark/settings-sharp.499x512.png</normaloff>:/darkicon/icons/dark/settings-sharp.499x512.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>48</width>
                 <height>48</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="HelpBtn">
               <property name="focusPolicy">
                <enum>Qt::ClickFocus</enum>
               </property>
               <property name="toolTip">
                <string>How to use the App</string>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>     Help</string>
               </property>
               <property name="icon">
                <iconset resource="../resources/resources.qrc">
                 <normaloff>:/darkicon/icons/dark/question-circle.512x512.png</normaloff>:/darkicon/icons/dark/question-circle.512x512.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>48</width>
                 <height>48</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
